spec/views/publications/edit.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
these are view testing which was also generated by scaffold. make sure you understand these also.